twitchio/websocket.py

```diff
@@ -283,6 +284,35 @@ def is_connected(self) -> bool:
             raise asyncio.TimeoutError(
                 f'Request to join the "{channel}" channel has timed out. Make sure the channel exists.')
 
+    async def part_channels(self, *channels: str):
+        """|coro|
+
+        Attempt to part the provided channels.
+
+        Parameters
+        ------------
+        *channels : str
+            An argument list of channels to attempt parting.
+        """
+
+        await asyncio.gather(*[self._part_channel(x) for x in channels])
+
+    async def _part_channel(self, entry):
+        channel = re.sub('[#\s]', '', entry).lower()
+
+        if channel not in self._channel_cache:
+            raise ClientError(f'Request to leave the "{channel}" channel failed. You are not in that channel.')
```

Comment:
Should be `Request to leave the channel "{channel}" failed.`

Reply:
Same for else where.

Reply:
Changed in 6a1feb1